scripts/modules/main.py

```diff
@@ -0,0 +1,116 @@
+import os
+import sys
+
+from .file_handler import collect_files, get_file_info, read_file_content, setup_console_encoding
+from .excel_handler import create_workbook, get_status_fills, write_file_info, write_file_sizes, write_markdown_consistency, write_formula_text, adjust_column_widths, save_workbook, print_statistics
+from .markdown_handler import compare_markdown_syntax_trees
+
+
+# Default directory and output file path
+source_dir = "g:\\temp\\defold_doc\\docs\\source"
+target_dir = "g:\\temp\\defold_doc\\docs\\target"
```

Comment:
Let's avoid hardcoding a default path which will rarely work for anyone but a few users. Either require the user to specify a path or use the same current working dir or similar